README.md

```diff
@@ -35,6 +35,12 @@ them, are:
   at the top of the new issue / PR page instead of that link. Maintainers can
   choose to inline content there and / or link to other pages as appropriate.
 
+And this is only the beginning. [@isaacs](https://github.com/isaacs/github) and
+the community collected a list of currently 443 issues that we are facing. Not 
```

Comment:
Would be better if instead of using the exact number as of today, it was a little more vague, like perhaps "over 400 issues and counting"